sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsExtensions.cs

```diff
@@ -67,9 +67,36 @@ public static class LiveMetricsExtensions
                 }
 
                 // INITIALIZE INTERNALS
-                var manager = new Manager(exporterOptions, new DefaultPlatform());
+                var manager = ManagerFactory.Instance.Get(exporterOptions);
                 return new LiveMetricsActivityProcessor(manager);
             });
         }
+
+        /// <summary>
+        /// TODO: Add documentation.
+        /// </summary>
+        /// <param name="loggerOptions"></param>
+        /// <param name="configure"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static OpenTelemetryLoggerOptions AddLiveMetrics(
+            this OpenTelemetryLoggerOptions loggerOptions,
+            Action<LiveMetricsExporterOptions> configure = null)
+        {
+            if (loggerOptions == null)
+            {
+                throw new ArgumentNullException(nameof(loggerOptions));
+            }
+
+            return loggerOptions.AddProcessor(sp =>
+            {
+                var options = new LiveMetricsExporterOptions();
+                configure?.Invoke(options);
+
+                // INITIALIZE INTERNALS
+                var manager = ManagerFactory.Instance.Get(options);
```

Comment:
We should use ServiceCollections rather than getting an instance from singleton. This way we could avoid the complete ManagerFactory. Please note live metrics support is only available through distro.